src/webapi/touch.rs

```diff
@@ -0,0 +1,215 @@
+use webapi::event_target::EventTarget;
+use webcore::try_from::{
+    TryFrom,
+    TryInto,
+};
+use webcore::value::{
+    ConversionError,
+    Reference,
+    Value,
+};
+
+/// The Touch interface represents a single contact point on a touch-sensitive
+/// device. The contact point is commonly a finger or stylus and the device may
+/// be a touchscreen or trackpad.
+///
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Touch)
+#[derive(Clone, Debug, Eq, PartialEq, ReferenceType)]
```

Comment:
Please put this specs link here:

https://w3c.github.io/touch-events/#touch-interface

You don't have to add links to the methods as it's not possible to link to the fields anyway.